docs/reference/search/suggesters.asciidoc

```diff
@@ -85,27 +91,35 @@ in the suggest text and if found an arbitrary number of options.
 --------------------------------------------------
 {
   "_shards": ...
-  "my-suggest-1": [ {
-    "text": "tring",
-    "offset": 0,
-    "length": 5,
-    "options": [ {"text": "trying", "score": 0.8, "freq": 1 } ]
-  }, {
-    "text": "out",
-    "offset": 6,
-    "length": 3,
-    "options": []
-  }, {
-    "text": "elasticsearch",
-    "offset": 10,
-    "length": 13,
-    "options": []
-  } ],
-  "my-suggest-2": ...
+  "hits": ...
+  "took": ...
+  "timed_out": ...
```

Comment:
I'd just say `"timed_out": false,` and `"took": <whatever number it take on the first run on your laptop>,`. I like `...` for when you are eliding large structures but for short stuff like this I'd just let it be an example. You still need the `s///` clause for `took` but I think it makes the docs easier to read if the reader doesn't have to guess what expected values for `took` are.

You could also do something like:
```
  ...,
  "suggest": {
  }
```
To elide the whole search response. I don't think that is a particularly good idea here because it hides that this is a search response. But it is a valid strategy. You could do it with `// TESTRESPONSE [s/...,/"hits": "$body.hits", "took": "$body.took", "timed_out": "$body.timed_out"/]`.